src/theory/datatypes/theory_datatypes.h

```diff
@@ -42,7 +42,7 @@ namespace datatypes {
 class TheoryDatatypes : public Theory {
  private:
   typedef context::CDList<Node> NodeList;
-  typedef context::CDHashMap<Node, int, NodeHashFunction> NodeIntMap;
+  typedef context::CDHashMap<Node, unsigned, NodeHashFunction> NodeUIntMap;
```

Comment:
If we are already changing the type, let's change it to a type of a fixed bit-width (`uint64_t`/`uint32_t`).

Reply:
This maps Nodes to an index in a vector. I'll change to size_t.